Week2/todo_app/program.js

```diff
@@ -4,69 +4,195 @@
 const mysql = require('mysql');
 
 class TodoModel {
-    constructor(dbConnection) {
-        this.dbConnection = dbConnection;
-    }
+  constructor(dbConnection) {
+    this.dbConnection = dbConnection;
+  }
 
-    // Loads all the TODOs in the database
-    load(callback) {
-        const selectTodoItems = "SELECT * FROM todo_items";
-        this.dbConnection.query(selectTodoItems, function(err, results, fields) {
-            if(err) {
-                callback(err);
-                return;
-            }
-
-            callback(null, results);
-        });
-    }
+  // Loads all the TODOs in the database
+  load(callback) {
+    const selectTodoItems = "SELECT * FROM todo_items";
+    this.dbConnection.query(selectTodoItems, function (err, results, fields) {
+      if (err) {
+        callback(err);
+        return;
+      }
 
-    create(description, callback) {
-        // Write code and query to create a new TODO item
-    }
+      callback(null, results);
+    });
+  }
 
-    update(id, description, callback) {
-        // Write code and query to update and existing TODO item
-    }
+  create(description, callback) {
+    // Write code and query to create a new TODO item
+    const createTodoItem = `INSERT INTO todo_items(text, is_completed, user_id) 
+    VALUES (${dbConnection.escape(description)}, 0, ${dbConnection.escape(arg2)} )`;
```

Comment:
Better to use `id` from your input as `user_id`. This works now because you defined a global variable `arg2` on line 107, but this is not a good practice if you want to change they way you name your variables one day in the future.